inlang/source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx

```diff
@@ -196,39 +195,45 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 		setSearchParams({ key: "lint", value: filteredMessageLintRules() })
 	})
 
-	const [localStorage, setLocalStorage] = useLocalStorage() ?? []
+	const [localStorage] = useLocalStorage() ?? []
+
+	// get lix errors
+	const [lixErrors, setLixErrors] = createSignal<ReturnType<Repository["errors"]>>([])
+	createEffect(() => {
+		// reset errors to empty on repo changes
+		setLixErrors([])
+
+		repo()?.errors.subscribe((errors) => {
+			setLixErrors(errors)
+		})
+	})
```

Comment:
Why is this needed. Can't you use repo().errors directly where you need it?

Reply:
Then no errors are detected at a later time because the subscription is missing.

Reply:
Is repo.errors() not reactive like project.errors()?

Reply:
According to @NilsJacobsen: Lix needs to be wrapped in a solid adapter with `from` so that signals can be used. Subscribe works without it. Otherwise a helper function is needed.